File: Day 041/492. Construct the Rectangle.java
class Solution {
    public int[] constructRectangle(int area) {
        int m = (int) Math.sqrt(area);
        int n = 0;
        for (int i = m; i > 0; i--) {
            if (area % i == 0) {
                n = i;
                break;
            }
        }
        int[] a = new int[2];
        a[0] = area / n;
        a[1] = n;
        return a;
    }
}

File: Day 041/492. Construct the Rectangle.py
class Solution(object):
    def constructRectangle(self, area):
        m = int(math.sqrt(area))
        for i in range (m,0,-1):
            if (area % i ==0):
                return [area/i,i]
        """
        :type area: int
        :rtype: List[int]
        """
        """
        :type area: int
        :rtype: List[int]
        """